src/styles/blocks/card.scss

```diff
@@ -0,0 +1,123 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  padding: 32px 0 16px;
+  width: 200px;
+  box-sizing: border-box;
+  display: block;
+  background-color: $bodyColor;
+  border-radius: 5px;
+  border: 1px solid $whiteGrayColor;
+}
+
+.card__img {
+  margin-left: 19px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__info {
+  height: 36px;
+  width: 166px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: $nameTextColor;
+  margin-top: 36px;
+  margin-left: 16px;
+  margin-right: 16px;
+  h3 {
```

Comment:
- You can use the shorthand property `margin`
- Add a blank line between code blocks

Fix similar things in other places